test/src/unit-cppapi-array.cc

```diff
@@ -614,7 +610,8 @@ TEST_CASE("C++ API: Encrypted array", "[cppapi], [encryption]") {
 }
 
 TEST_CASE(
-    "C++ API: Encrypted array, std::string key", "[cppapi], [encryption]") {
+    "C++ API: Encrypted array, std::string key",
+    "[cppapi][encryption][cppapi-encryption]") {
```

Comment:
tags like `[cppapi-encpytion]` are redundant as they express the same thing as specifying two tags to select unit tests over

Reply:
We will address this in a subsequent PR, where we will also tide up / refactor some of the tests.